src/MapPanel.tsx

```diff
@@ -29,10 +31,10 @@ export class MapPanel extends Component<MapPanelProps> {
     };
     this.lastOptions = {...this.props.options};
     this.theme = createTheme();
-    let self = this;
+    // let self = this;
     PubSub.subscribe('returnMapCenterAndZoom', this.updateCenter);
     PubSub.subscribe('returnMapViewport', this.updateMapViewport);
-    PubSub.subscribe('topologyRefresh', ()=>{ self.updateMap() });
+    PubSub.subscribe('topologyRefresh', ()=>{ this.updateMap() });
```

Comment:
Do any of our end to end tests cover this specific line? Just making sure that the `this` here resolves properly. As a reminder, this code is not covered by any of the unit tests.

Reply:
In the absence of MapPanel React component testing, I'm restoring this to the original code until we can get such a framework in place.